templates/home/reports/pre-a11y-1.json

```
@@ -0,0 +1,444 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/templates/home/home-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Assessing the main content of the GC Home Page template",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": "#wb-main"
```

This selector return nothing in that page.

In fact we would have 2 CSS selector because you may didn't know that the top menu button color is different compared to other regular page. And I am adding another CSS selector to include the items of appearing with the menu button because that would be evaluated along with the gcweb-menu plugin.

```suggestion
      "@value": "main, .gcweb-menu button"
```